configs/default/HGLR-HGLRCF411SX1280.config

```diff
@@ -86,6 +86,7 @@ dma pin B07 0
 
 # feature
 feature RX_SPI
+feature -RX_SERIAL
```

Comment:
We can leave this out as we already use feature RX_SPI

Reply:
Serial Rx is set as default in the new build system for 4.4. This is currently needed to default to SPI RX. Otherwise serial rx is preselected after applying custom defaults.

Reply:
You also can define:

    #define DEFAULT_RX_FEATURE              FEATURE_RX_SPI
    #define RX_SPI_PROTOCOL                 EXPRESSLRS

Reply:
Yes i know, but i think since it's all overhauled for 4.5 and i will make a PR for this board there too, this is enough for now.